src/content/guide/how-to-build-a-product/authentication.md

```diff
@@ -0,0 +1,423 @@
+
+Integral to OAuth is the concept of *clients*, and *client credentials*.
+An OAuth client generally represents an application, like a native iOS
+application running on an iPhone or a web app running in a browser. As
+part of the setup process for your product, you will need to create an
+OAuth client for your organization.
+
+Your organization's OAuth client will be needed when hitting the
+Particle API to perform actions that only your organization's team
+members or applications should be able to perform. An example of this is
+creating a customer that belongs to your organization via the Particle
+API. Only those with a valid OAuth client will be able to perform this
+action, protecting your organization from fake or erroneous new customer
+accounts.
+
+*Client credentials* are comprised of two pieces of information: A
+**client ID**, and a **secret**. Passing these two pieces of data to
+the Particle API will allow you to create **access tokens**, which are
+described in detail in the next section.
+
+### Scopes
+
+Scopes allow you to specify exactly what type of access the client (and
+tokens created using the client) should have. Scopes are used for
+security reasons to limit the allowed applications of OAuth client
+credentials. Depending on which authentication method you choose, you
+may need to specify a scope when creating your OAuth client. [More on
+scopes](http://tools.ietf.org/html/rfc6749#section-3.3).
+
+### Creating an OAuth Client
+
+Soon (Late August 2015) you will be able to go to the
+[Particle dashboard](https://dashboard.particle.io/) and click
+“Create app” to create a client ID and secret.
+(Ping @zachary in the [forum](http://community.particle.io/)
+if we fall behind on this estimate or you just want to say how much
+you want this feature!)
+
+![Create OAuth Client](/assets/images/create-oauth-client.png)
+<p class="caption">Use the dashboard to create your OAuth client
+credentials</p>
+
+Right now you can make an API call to create an create an OAuth client using the
+instructions in our [API reference](/reference/api/#create-an-oauth-client)
+Your web app will use the generated client ID and secret  when you hit the
+Particle API on behalf of your organization and your customers to manage all the
+devices in your product fleet.
+
+In the API reference, note how there is an optional parameter called
+`organization`. In order to create client credentials for your
+organization, you should pass the **slug** of your organization as this
+parameter. The easiest way to find the slug of your organization is in
+the URL address when viewing your organization's dashboard, just after
+the `/` in `dashboard.particle.io/`. Example:
+`dashboard.particle.io/my-org`, where `my-org` is the organization slug.
+The API request to create client credentials scoped to your organization
+will look like this:
+
+```bash
+$ curl -d name=MyApp -d type=installed -d organization:my-org -d access_token=1234
```

Comment:
Also add `-X POST` because on linux I've heard that the `-d` flag doesn't necessarily set the default verb to post.